proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -64,36 +65,37 @@ The top-level container contains a single key that matches the name of the enum
 which points to another container that contains the values as they would be encoded
 for structs and classes.
 
-Associated values can also be unlabeled, in which case they will be encoded into an
-array instead, or just the raw value if only one parameter is present.
+Associated values can also be unlabeled, in which case an identifier will be generated in the form of `_$N`, where `$N` is the 0-based position of the parameter.
 
 ```swift
 enum Command: Codable {
   case load(String)
-  case store(String, value: Int)
+  case store(key: String, Int)
 }
 ```
 
 would encoded to
 
 ```json
 {
-  "load": "MyKey"
+  "load": {
+    "_0": "MyKey"
+  }
 }
 ```
 
 and 
 
 ```json
 {
-  "store": [
-    "MyKey",
-    42
-  ]
+  "store": {
+    "key": "MyKey",
+    "_1": 42
+  }
 }
 ```
 
-An enum case without associated values would be encoded as `true` value.
+An enum case without associated values would be encoded as an empty `KeyedContainer`,
```

Comment:
A note indicating that this allows an easier time with shape changes would be nice.